### ISSUE TITLE
rand_vms.c mod for VMS V8.3 and earlier

### ISSUE THREAD

**Issue by mgdaniel** [openssl/openssl#8487]
[.openssl-1_1_1b.crypto.rand]rand_vms.c

rand_pool_add_nonce_data()
```
#if __CRTL_VER >= 80400000
    sys$gettim_prec(&data.time);
#else
    sys$gettim((void*)&data.time);
#endif
```
need to be extended to the use of
```
sys$gettim_prec(&data.time);
```
in rand_pool_add_additional_data()

**Comment by mspncp** [comment]
Strange. Why hasn't this been noticed when your issue #7229 ws fixed by pull request #7230?

Hang on for a second, fix is on the way...

**Comment by mspncp** [comment]
Please try #8488 and let me know whether it fixes your issue.
